LCM/dsc/engine/ca/CAInfrastructure/CAEngine.c

```diff
@@ -1099,76 +1098,177 @@ MI_Result MI_CALL SendConfigurationApply( _In_ LCMProviderContext *lcmContext,
     CleanupResourceErrorList(&resourceErrorList);
     if( r != MI_RESULT_OK )
     {
-        FreeExecutionOrderContainer(&executionContainer);            
-        MI_Session_Close(&miSession, NULL, NULL);            
-        return r;          
+        FreeExecutionOrderContainer(&executionContainer);
+        MI_Session_Close(&miSession, NULL, NULL);
+        return r;
     }
 
-    FreeExecutionOrderContainer(&executionContainer);    
-    MI_Session_Close(&miSession, NULL, NULL);   
+    FreeExecutionOrderContainer(&executionContainer);
+    MI_Session_Close(&miSession, NULL, NULL);
     return r;
 }
 
-MI_Result GetCurrentState(_In_ ProviderCallbackContext *provContext,  
+MI_Result GetCurrentState(_In_ ProviderCallbackContext *provContext,
                            _In_ MI_Application *miApp,
                            _In_ MI_Session *miSession,
                            _In_ MI_Instance *instance,
                            _In_ const MI_Instance *regInstance,
-                           _Outptr_result_maybenull_ MI_Instance **outputInstance,
+                           _Outptr_result_maybenull_ MI_Instance *outputInstance,
                            _Outptr_result_maybenull_ MI_Instance **extendedError)
 {
-    *outputInstance = NULL;
-
-    if( Tcscasecmp(regInstance->classDecl->name, BASE_REGISTRATION_WMIV2PROVIDER) == 0 ||
-        Tcscasecmp(MSFT_LOGRESOURCENAME, instance->classDecl->name) == 0)
+    if (
+#if !defined(BUILD_OMS)
+        Tcscasecmp(regInstance->classDecl->name, BASE_REGISTRATION_WMIV2PROVIDER) == 0 ||
```

Comment:
>Tcscasecmp(regInstance->classDecl->name, BASE_REGISTRATION_WMIV2PROVIDER) == 0 [](start = 8, length = 78)

should we not ignore it if g_DscHost == FALSE?

Reply:
existing behavior. So I'm gonna keep it.

---
In reply to: [276397846](https://github.com/Microsoft/PowerShell-DSC-for-Linux/pull/553#discussion_r276397846) [](ancestors = 276397846)